src/transformers/modeling_attn_mask_utils.py

```diff
@@ -188,6 +188,8 @@ def _expand_mask(mask: torch.Tensor, dtype: torch.dtype, tgt_len: Optional[int]
     @staticmethod
     def _unmask_unattended(
         expanded_mask: torch.FloatTensor,
+        attention_mask: torch.FloatTensor,
+        input_tensor: torch.FloatTensor,
```

Comment:
I don't think we need `input_tensor`, do we? The FX check should be doable on `attention_mask`

Reply:
but sometimes the attention mask is None even when tracing no?

Reply:
I meant `expanded_mask`. See the typing

Reply:
alright